package.json

```diff
@@ -77,16 +77,16 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^9.1.0",
+    "@angular/animations": "^11.2.5",
     "@angular/cdk": "^8.2.3",
-    "@angular/common": "^9.1.0",
-    "@angular/compiler": "^9.1.0",
-    "@angular/core": "^9.1.0",
+    "@angular/common": "^11.2.5",
+    "@angular/compiler": "^11.2.5",
+    "@angular/core": "^11.2.5",
     "@angular/fire": "5.2.1",
-    "@angular/forms": "^9.1.0",
-    "@angular/platform-browser": "^9.1.0",
-    "@angular/platform-browser-dynamic": "^9.1.0",
-    "@angular/router": "^9.1.0",
+    "@angular/forms": "^11.2.5",
+    "@angular/platform-browser": "^11.2.5",
+    "@angular/platform-browser-dynamic": "^11.2.5",
+    "@angular/router": "^11.2.5",
     "@fontsource/roboto": "4.2.1",
     "@ionic/angular": "^5.5.2",
```

Comment:
Small bump, but probably good to do also. @ionic/angular is at `5.6.0`

Reply:
@flemcito I would wait with that as to not rock the boat more than necessary - otherwise you won't know whether any regressions are a result of bumping Angular or bumping Ionic... (I agree we should do it soonish, but I don't agree on doing it as part of this PR)